### PR TITLE
{lib}[foss/2018b] pyFFTW v0.11.1 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by arielzn**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/pyFFTW/pyFFTW-0.11.1-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+]
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['05ea28dede4c3aaaf5c66f56eb0f71849d0d50f5bc0f53ca0ffa69534af14926']
```
Comment (lexming) [inline easybuild/easyconfigs/p/pyFFTW/pyFFTW-0.11.1-foss-2018b-Python-3.6.6.eb]: Please move this part between `toolchain` and `dependencies`. It is not a strict error but we try to keep some common structure between the easyconfigs.

File: easybuild/easyconfigs/p/pyFFTW/pyFFTW-0.11.1-foss-2018b-Python-3.6.6.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(namelower)s'],
+}
```
Comment (lexming) [inline easybuild/easyconfigs/p/pyFFTW/pyFFTW-0.11.1-foss-2018b-Python-3.6.6.eb]: This `sanity_check_paths` are default with `PythonPackage`. They can be removed.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/6f2317dcffa5b5c7a1514d76be5483a4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/650cd09acee8581eb7af32fc2583724c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @arielzn!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/cf86d2513545592bdeeaccd19513ab18 for a full test report.

**Pull request merged**